src/app/hello/hello.js

```diff
@@ -0,0 +1,6 @@
+import template from './hello.html';
+import './hello.css';
+const AppComponent = {
+    template
+};
+export default AppComponent;
```

Comment:
And components like this

![image](https://user-images.githubusercontent.com/37654233/57135462-c8f65c00-6da9-11e9-962c-c669a85a67a4.png)